python-sdk/src/astro/dataframes/load_options.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import annotations
+
+from attr import define
+from pandas._typing import DtypeArg
+
+from astro.options import LoadOptions
+
+
+@define
+class PandasLoadOptions(LoadOptions):
+    pass
+
+
+@define
+class CsvLoadOptions(PandasLoadOptions):
+    delimiter: str | None = None
+    dtype: DtypeArg | None = None
+
+
+@define
+class JsonLoadOptions(PandasLoadOptions):
+    encoding: str | None = None
+
+
+@define
+class NdjsonLoadOptions(PandasLoadOptions):
+    ndjson_normalize_sep: str = "_"
+
+
+@define
+class ParquetLoadOptions(PandasLoadOptions):
+    columns: list[str] | None = None
```

Comment:
For now, we should prefix all of these with Pandas - once we have a generic representation of these, which works across DBs & pandas, then we can give a generic name and place them together with the file file types.